spec/factories/locations.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```

Comment:
It might be worth looking at how some grid references [were generated in digital-services-core](https://github.com/EnvironmentAgency/digital-services-core/blob/develop/spec/factories/factories.rb), rather than using the same grid reference in all instances.